EDEngineer/Resources/Data/blueprints.json

```diff
@@ -14204,6 +14204,158 @@
     "Grade": 5
   },
   {
+    "Type": "Life Support",
+    "Name": "",

```

Comment:
Name should be Refill, maybe 100% Refill ?
Empty string won't show nicely in the blueprint list